src/App.vue

```diff
@@ -131,16 +156,35 @@ export default {
       .then(response => {
         this.beers = response.data;
 
-        this.beers = this.beers.map(beer => ({
-          id: beer.id,
-          name: beer.name,
-          tagline: beer.tagline,
-          description: beer.description,
-          image_url: beer.image_url,
-          abv: beer.abv,
-          food_pairing: beer.food_pairing,
-          isFavourite: false
-        }));
+        // If we have favourite beers in localStorage, retrieve them.
+        if (localStorage.getItem('beerFavourites')) {
+          try {
+            this.myFavourites = JSON.parse(
+              localStorage.getItem('beerFavourites')
+            );
+          } catch (e) {
+            localStorage.removeItem('beerFavourites');
+          }
+        }
+
+        this.beers = this.beers.map(beer => {
+          // Is the current beer in my favourites array?
+          const beerInFavourites = !!this.myFavourites.find(
+            element => element === beer.id
+          );
+
+          // Beer object.
+          return {
+            id: beer.id,
+            name: beer.name,
+            tagline: beer.tagline,
+            description: beer.description,
+            image_url: beer.image_url,
+            abv: beer.abv,
+            food_pairing: beer.food_pairing,
+            isFavourite: beerInFavourites ? true : false
```

Comment:
this is the same as `isFavourite: beerInFavourites`

which could even be 
```
const isFavourite = !!this.myFavourites.find(
            element => element === beer.id
          );

 return {
            id: beer.id,
            name: beer.name,
            tagline: beer.tagline,
            description: beer.description,
            image_url: beer.image_url,
            abv: beer.abv,
            food_pairing: beer.food_pairing,
            isFavourite,
};
```

Reply:
Really nice :)